<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowServer</class>
 <widget class="QMainWindow" name="MainWindowServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowServer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonSoundReset">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сброс звука</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAlarmReset">
        <property name="text">
         <string>Сброс тревог</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSoundAlarm">
        <property name="text">
         <string>Звук &quot;Тревога&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>38</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>write</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Дерево объектов</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSelectedUN">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBarDKWait">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Журнал событий</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>18</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>18</number>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Причины и принятые меры</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelReason">
             <property name="text">
              <string>Причины</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="labelTakenMeasures">
             <property name="text">
              <string>Принятые меры</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="comboBoxReason">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="toolButtonReason">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="comboBoxTakenMeasures">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QToolButton" name="toolButtonTakenMeasures">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Диагностика</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>18</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuJournal">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Журнал</string>
    </property>
    <widget class="QMenu" name="menuFontSize">
     <property name="title">
      <string>Размер шрифта в журнале</string>
     </property>
     <addaction name="actionIncrease"/>
     <addaction name="actionReduce"/>
    </widget>
    <addaction name="actionNewScheme"/>
    <addaction name="separator"/>
    <addaction name="actionDataBase"/>
    <addaction name="separator"/>
    <addaction name="menuFontSize"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSignaling">
    <property name="title">
     <string>Сигнализация</string>
    </property>
    <addaction name="actionSoundReset"/>
    <addaction name="separator"/>
    <addaction name="actionAlarmReset"/>
    <addaction name="separator"/>
    <addaction name="actionSoundAlarm"/>
   </widget>
   <widget class="QMenu" name="menuManagement">
    <property name="title">
     <string>Управление</string>
    </property>
    <widget class="QMenu" name="menuDK">
     <property name="title">
      <string>ДК</string>
     </property>
     <widget class="QMenu" name="menuRifDK">
      <property name="title">
       <string>РИФ</string>
      </property>
      <addaction name="actionRifDKOverall"/>
     </widget>
     <widget class="QMenu" name="menuSsoiDK">
      <property name="title">
       <string>ССОИ</string>
      </property>
      <addaction name="actionSsoiDKOverall"/>
      <addaction name="actionSsoiDKCh1"/>
      <addaction name="actionSsoiDKCh2"/>
      <addaction name="actionSsoiDKCh3"/>
      <addaction name="actionSsoiDKCh4"/>
     </widget>
     <addaction name="menuRifDK"/>
     <addaction name="menuSsoiDK"/>
    </widget>
    <addaction name="actionControl"/>
    <addaction name="separator"/>
    <addaction name="menuDK"/>
    <addaction name="separator"/>
    <addaction name="actionDiagnostics"/>
    <addaction name="separator"/>
    <addaction name="actionCustomization"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuJournal"/>
   <addaction name="menuSignaling"/>
   <addaction name="menuManagement"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewScheme">
   <property name="text">
    <string>Новая схема</string>
   </property>
  </action>
  <action name="actionDataBase">
   <property name="text">
    <string>База данных</string>
   </property>
   <property name="iconText">
    <string>База данных</string>
   </property>
   <property name="toolTip">
    <string>База данных</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionSoundReset">
   <property name="text">
    <string>Сброс звука</string>
   </property>
  </action>
  <action name="actionAlarmReset">
   <property name="text">
    <string>Сброс тревог</string>
   </property>
  </action>
  <action name="actionSoundAlarm">
   <property name="text">
    <string>Звук &quot;Тревога&quot;</string>
   </property>
  </action>
  <action name="actionControl">
   <property name="text">
    <string>Контроль</string>
   </property>
  </action>
  <action name="actionDiagnostics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Диагностика</string>
   </property>
  </action>
  <action name="actionCustomization">
   <property name="text">
    <string>Настройка</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе ...</string>
   </property>
  </action>
  <action name="actionIncrease">
   <property name="text">
    <string>Увеличить</string>
   </property>
  </action>
  <action name="actionReduce">
   <property name="text">
    <string>Уменьшить</string>
   </property>
  </action>
  <action name="actionRifDKOverall">
   <property name="text">
    <string>Общий</string>
   </property>
  </action>
  <action name="actionSsoiDKOverall">
   <property name="text">
    <string>Все каналы</string>
   </property>
  </action>
  <action name="actionSsoiDKCh1">
   <property name="text">
    <string>Канал 1</string>
   </property>
  </action>
  <action name="actionSsoiDKCh2">
   <property name="text">
    <string>Канал 2</string>
   </property>
  </action>
  <action name="actionSsoiDKCh3">
   <property name="text">
    <string>Канал 3</string>
   </property>
  </action>
  <action name="actionSsoiDKCh4">
   <property name="text">
    <string>Канал 4</string>
   </property>
  </action>
  <action name="actionExpandUNTree">
   <property name="text">
    <string>Развернуть дерево объектов</string>
   </property>
   <property name="toolTip">
    <string>Развернуть дерево объектов</string>
   </property>
  </action>
  <action name="actionCollapseUNTree">
   <property name="text">
    <string>Свернуть дерево объектов</string>
   </property>
   <property name="toolTip">
    <string>Свернуть дерево объектов</string>
   </property>
  </action>
  <action name="actionOnOff">
   <property name="text">
    <string>Включить/Выключить</string>
   </property>
   <property name="toolTip">
    <string>Включить/Выключить</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="toolTip">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="actionDK">
   <property name="text">
    <string>ДК</string>
   </property>
   <property name="toolTip">
    <string>ДК</string>
   </property>
  </action>
  <action name="actionRemoteControlOn">
   <property name="text">
    <string>Работа с пультом</string>
   </property>
   <property name="toolTip">
    <string>Работа с пультом</string>
   </property>
  </action>
  <action name="actionAllOff">
   <property name="text">
    <string>Выключить все</string>
   </property>
   <property name="toolTip">
    <string>Выключить все</string>
   </property>
  </action>
  <action name="actionRemoteControlOff">
   <property name="text">
    <string>Завершение работы с пультом</string>
   </property>
  </action>
  <action name="actionGuardStamp">
   <property name="text">
    <string>Отметка часового</string>
   </property>
  </action>
  <action name="actionConnectBlock_pdi">
   <property name="text">
    <string>ConnectBlock_pdi</string>
   </property>
  </action>
  <action name="actionNormUN">
   <property name="text">
    <string>Норма</string>
   </property>
  </action>
  <action name="actionAlarmUN">
   <property name="text">
    <string>Тревога (сработка)</string>
   </property>
  </action>
  <action name="actionNoConnectUN">
   <property name="text">
    <string>Нет связи</string>
   </property>
  </action>
  <action name="actionUNOn">
   <property name="text">
    <string>Включить</string>
   </property>
  </action>
  <action name="actionUNOff">
   <property name="text">
    <string>Выключить</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
